tests/test-team/template-mgmt-component-tests/template-mgmt-choose-page.component.ts

```diff
@@ -169,3 +169,180 @@ test.describe('Choose Template Type Page', () => {
     await expect(chooseTemplatePage.goBackLink).toBeHidden();
   });
 });
+
+test.describe('NHS App Message Template tests -in progress', () => {
+  const sessionStorageHelper = new SessionStorageHelper([
+    emptySessionData,
+    emptySessionDataForRadioSelect,
+    nhsAppRadioSelectedSessionData,
+  ]);
+
+  test.beforeAll(async () => {
+    await sessionStorageHelper.seedSessionData();
+  });
+
+  test.afterAll(async () => {
+    await sessionStorageHelper.deleteSessionData();
+  });
+  test('Validate error messages on the create NHS App message template page', async ({
```

Comment:
all these tests are being added in 'template-mgmt-choose-page' but they don't appear to have anything to do with the choose page?

Reply:
Tests moved create nhs app template